codespell_lib/data/dictionary.txt

```diff
@@ -26500,16 +26584,37 @@ peacd->peace
 peacefuland->peaceful and
 peacify->pacify
 peageant->pageant
+peanochle->pinochle
+peanockle->pinochle
+peanuchle->pinochle
+peanuckle->pinochle
+peapel->people
+peapels->peoples
 peaple->people
 peaples->peoples
+pease->peace, piece,
```

Comment:
and `please`, `lease`

Same for derived words below, when applicable.

Reply:
Added.